code/modules/spells/targeted/projectile/fireball.dm

```diff
@@ -36,6 +36,13 @@
 	explosion(get_turf(spell_holder), ex_severe, ex_heavy, ex_light, ex_flash)
 	return targets
 
+/spell/targeted/projectile/dumbfire/fireball/choose_prox_targets(mob/user = usr, var/atom/movable/spell_holder)
+	var/list/targets = ..()
+	for(var/mob/living/M in targets)
+		if(M.isUnconscious())
```

Comment:
Does this check if they're resting?
Fireballs should pass over people lying down, imo.